packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -72,15 +69,15 @@ export default VTextField.extend({
     itemDisabled: {
       type: [String, Array, Function],
       default: 'disabled'
-    },
+    } as PropValidator<string | (string | number)[] | ((item: object, fallback?: any) => any)>,
```

Comment:
define a type for `string | (string | number)[] | ((item: object, fallback?: any) => any)` to avoid duplication

Reply:
@jacekkarczmarczyk how should I name this?